t/044-req-body.t

```diff
@@ -159,11 +159,8 @@ hiya, world"]
 hello, world",
 "POST /bar
 hiya, world"]
---- response_body eval
-["body: nil\n",
-qr/400 Bad Request/]
```

Comment:
It's bad to remove existing checks from the test case. We should fix the checks instead. I'll do that.